src/dndbeyond/base/utils.js

```diff
@@ -154,6 +154,18 @@ function buildAttackRoll(character, attack_source, name, description, properties
                     crit_damage_types.push(damage_types[i]);
                 }
             }
+            if (character.hasFeat("Piercer")) {
+                for (i = 0; i < damage_types.length; i++) {
+                    if (damage_types[i].includes("Piercing")){
+                        let piercer_damage = damagesToCrits(character, [damages[i]]);
+                        if (piercer_damage.length > 0 && piercer_damage[0] != "") {    
+                            piercer_damage[0] = piercer_damage[0].replace(/([0-9]+)d([0-9]+)/, '1d$2');
+                            crit_damages.push(piercer_damage[0]);
+                            crit_damage_types.push("Piercer Feat");
+                        }
+                    }
+                }
```

Comment:
missing a `break;` in the for loop since it says you can reroll one of the attack's damage dice.
Will fix post merge

Reply:
Ah yeah, good catch.

Real fun trying to find a weapon thing with multiple piercing damages...